client/my-sites/backup/backup-contents-page/file-browser/use-backup-path-info-query.ts

```diff
@@ -32,8 +33,13 @@ export const useBackupPathInfoQuery = (
 		select: parseBackupPathInfo,
 		staleTime: Infinity,
 		retry: 2,
-		onError: () => {
-			dispatch( onRetrievingFileInfoError() );
-		},
 	} );
+
+	useEffect( () => {
+		if ( query.isError ) {
```

Comment:
We generally recommend handling errors on the consumer component side of things if it's not an implementation detail of the custom query hook.